content/influxdb/v2.0/reference/flux/stdlib/profiler/_index.md

```diff
@@ -56,14 +58,18 @@ include a table with the following columns:
 - **influxdb/scanned-values**: value scanned by InfluxDB.
 - **influxdb/scanned-bytes**: number of bytes scanned by InfluxDB.
 
-#### Use the query profiler
-
-Use the query profiler to output statistics about query execution.
-
-```js
-import "profiler"
-
-option profiler.enabledProfilers = ["query"]
+### operator
+The `operator` profiler output statistics about each operation in a query.
+[Pushdown operations](/influxdb/v2.0/query-data/optimize-queries/#start-queries-with-pushdown-functions)
+that execute in the data source storage engine are returned as a single operation.
+Operations executed in memory are returned as individual operations.
```

Comment:
> Operations executed in memory are returned as individual operations. 

This is a little misleading since the pushed down operations are also often executed in memory, it's just they are executed in the storage tier. You might write "operations executed in the storage tier" instead. (Feel free to disregard if you want to keep consistent terminology with elsewhere in the docs though)